common/views/components/IIIFViewerPrototype/GridViewerPrototype.tsx

```diff
@@ -86,7 +86,7 @@ const Cell = memo(({ columnIndex, rowIndex, style, data }: CellProps) => {
               clickHandler={() => {
                 mainViewerRef &&
                   mainViewerRef.current &&
-                  mainViewerRef.current.scrollToItem(itemIndex);
+                  mainViewerRef.current.scrollToItem(itemIndex, 'start');
```

Comment:
@davidpmccormick what is `scrolltoItem`?

Reply:
It's a method on react window's `FixedSizedList` (which we use for virtualisation): https://react-window.now.sh/#/api/FixedSizeList

By default it'll scroll as little as possible. The 'start' align option gets you to the top of the item, which is almost certainly what you'd want to happen.

Reply:
ah brilliant, sorry I haven't really looked at that lib.